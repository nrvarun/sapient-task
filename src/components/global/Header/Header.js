import style from "./header.module.scss";

import Link from "next/link";

const Header = () => {
  return (
    <header>
      <div className={style.content}>
        <Link href="/" as="/">
          <a>
            <h1
              data-test-id="logo"
              className={`fnt-700 text-capitalize ${style.title}`}
            >
              SpaceEx launch programs
            </h1>
          </a>
        </Link>
      </div>
    </header>
  );
};

export default Header;
