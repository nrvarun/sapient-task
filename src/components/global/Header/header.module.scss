.title {
  margin: 0 0 30px 0;
  padding: 15px 15px 0 15px;

  @media (min-width: 767px) {
    margin: 0 0 40px 0;
  }

  @media (min-width: 992px) {
    margin: 0 0 50px 0;
  }

  @media (min-width: 1200px) {
    margin: 0 0 60px 0;
  }
}
