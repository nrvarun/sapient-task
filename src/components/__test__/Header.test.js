import Header from "../global/Header";

describe("Header component", () => {
  const wrapper = shallow(<Header />);

  it("Should match snapshot", () => {
    expect(wrapper).toMatchSnapshot();
  });

  it("should have text", () => {
    const logo = wrapper.find("h1");
    expect(logo.length).toBe(1);
  });
});
