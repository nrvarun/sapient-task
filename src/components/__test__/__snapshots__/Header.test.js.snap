// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component Should match snapshot 1`] = `ShallowWrapper {}`;
